app/controllers/donations.js

```diff
@@ -228,9 +241,14 @@ module.exports = (app) => {
           DonationId: results.createDonation.id,
           amount: amountFloat,
           currency,
-          platformFee: applicationFee,
-          paymentProcessingFee: 0, // TODO: Need to make a separate call for this
-          data: charge,
+          fxCurrency: balanceTransaction.currency,
+          fxAmount: balanceTransaction.amount,
+          fxRate: amountInt/balanceTransaction.amount,
```

Comment:
Looks like 2% is the fxrate for Stripe
https://support.stripe.com/questions/how-do-currency-conversions-work-on-connect